extension.meta.xml

```diff
@@ -13,13 +13,16 @@
 		</author>
 	</authors>
 	<releases>
-		<release version="1.0" date="2011-03-17" min="2.2" max="2.2.5" />
-		<release version="1.1" date="2012-05-08" min="2.3" />
+		<release version="1.3" date="2016-05-17" min="2.3" max="2.6.7">
+			Updated `...nce()->Author->isD...` to `...nce()->Author()->isD...` per [migration guide](https://github.com/symphonycms/symphony-2/wiki/Migration-Guide-to-2.5-for-Developers#properties).
```

Comment:
max should be `"2.x.x"` as this is the best practice right now. The 2.x series will not break anything.